www/js/settings.json

```diff
@@ -124,7 +124,7 @@
             "options": {
               "sttm2": "STTM 2",
               "igurbani": "iGurbani",
-              "sttm": "STTM Living"
+              "sttm": "BaniDB Living"
```

Comment:
Needs to be updated to `HELP.md` as well

Reply:
done